Assets/Scripts/Animation/GhostFadeAnim.cs

```diff
@@ -3,10 +3,18 @@
 using UnityEngine;
 
 public class GhostFadeAnim : MonoBehaviour {
-    [SerializeField]
-    private float fadeRate = 2f;
-    [SerializeField]
-    private float fadeDelay = 2f;
+
+	[SerializeField]
```

Comment:
fadeRate and fadeDelay don't need to be serialized since we are setting them via script & no longer need to be able to access them through the inspector. (And it's better that we aren't able to access them in the inspector, since making changes there would have no effect.)